### PR TITLE
Issue/23/manager

### PR DESCRIPTION
**Pull request by m-aguena**
first working version of the manager.

### CODE REVIEW

File: clmm/core/manager.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+class Manager():
+
+    '''
+    Makes the interactions between GalaxyCluster and the other clmm objects
+    '''
+
+    def __init__(self):
+        '''
+        '''
+        pass
```
Comment (matthewkirby) [inline clmm/core/manager.py]: Should this `pass` be here? There is code after it

File: clmm/data__types.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+"""
```
Comment (matthewkirby) [inline clmm/data__types.py]: This file name has two underscores? Is this intended?

Comment (matthewkirby) [reply clmm/data__types.py]: It also has the same name as clmm/core/datatypes.py

Comment (matthewkirby) [reply clmm/data__types.py]: Actually, was the file just moved? What happened to the one in clmm/core?

Comment (m-aguena) [reply clmm/data__types.py]: @matthewkirby as far as I could find out, the file clmm/core/datatypes.py is the one being imported by all other modules. I dont know why clmm/data__types.py exists.
@combet do you know what is this purpose of this file? Is it actually being used? I could not find an import of it in any other part.

File: clmm/core/manager.py (hunk 1 above)
Comment (m-aguena) [reply clmm/core/manager.py]: removed

File: clmm/data__types.py (hunk 2 above)
Comment (cavestruz) [reply clmm/data__types.py]: @matthewkirby  -  I think all requested changes were made.  It looks like you have to approve the requested changes(?)

**Pull request merged**